### ISSUE TITLE
Angelscript code completion not working in later versions

### ISSUE THREAD

**Issue by cmann1** [eranif/codelite#2647]
I'm currently on 12.0.12 and code completion works for most things.
I haven't tried all the versions, but 12.0.16 and above seems to break it.

**Comment by eranif** [comment]
Can you provide a code example that does not work for you?

**Comment by cmann1** [comment]
Hmm I just installed the latest codelite again and it seems to be working better, even though I'm sure it wasn't working at all last time.

I'm not sure if this is specific to using angelscript syntax, but one thing I have noticed now though is that a class won't be recognised until I open the file it's declared in and modify and save it.

Another thing which might be related is that some classes are highlighted and other not:
![image](https://user-images.githubusercontent.com/4889816/106302665-eb105d00-6261-11eb-80ce-68bf61c5b776.png)
I haven't been able to isolate exactly why though.
(In this particular case code completion does work for both classes though)
